modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/resources/com/liferay/dynamic/data/mapping/internal/upgrade/v5_0_0/dependencies/ddm-form-instance-settings.json

```diff
@@ -5,6 +5,11 @@
 			"name": "autosaveEnabled",
 			"value": "true"
 		},
+		{
+			"fieldReference": "body",
```

Comment:
Do we need to rename since we changed the variables names?

Reply:
forgot about that sorry